mappings/net/minecraft/network/chat/MessageSigner.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/unmapped/C_byvkekfd net/minecraft/network/chat/MessageSigner
+	FIELD f_nyrtheqj timeStamp Ljava/time/Instant;
+	FIELD f_urmrfeuy salt J
```

Comment:
`salt`?

Reply:
Probably this if it's message signing - https://en.m.wikipedia.org/wiki/Salt_(cryptography)#:~:text=In%20cryptography%2C%20a%20salt%20is,to%20safeguard%20passwords%20in%20storage.

Reply:
Oh, interesting